### DISCUSSION TITLE
error 406

### DISCUSSION THREAD

**Discussion by IReena8** [community/community#21260] [Discussions] [answered]
Hello! How fix error 406?

![Screenshot_1](https://user-images.githubusercontent.com/36493429/180814131-c4c49531-076c-46c5-9d03-babbb512a665.png)

**Comment by MrSarno** [comment]
I’ve gone to that repo, and the URL does not seem correct;

https://github.com/IReena8/GitBash

Your URL expects a Postman directory; there is none.

**Reply by IReena8** [reply]
Oh thank you!

**Reply by IReena8** [reply]
What exactly should I make? Should I activate all the Branch Protection checkboxes?

**Reply by IReena8** [reply]
What exactly should I make? Should I activate all the Branch Protection checkboxes?

**Reply by ronaldkroon** [reply]
It seems direct links to files are broken.

**Reply by IReena8** [reply]
Can this be fixed somehow so that I can continue to work with these documents? Or do I need to create a new Github?

**Reply by MrSarno** [reply]
It seems as if the file(s) you’re looking for haven’t been pushed to GitHub.

Unless this happens to be what you want?

https://github.com/IReena8/GitBash/blob/main/HW_Postman_1.postman_collection.json

**Reply by MrSarno** [reply]
I’ve looked at your public repositories and I can’t see any with a Postman directory.

Do you have any private repositories you could look in?

**Reply by IReena8** [reply]
No) It's my files!

**Reply by IReena8** [reply]
I created them and then I sent link one of files other person

**Reply by IReena8** [reply]
Yes! I have
![Screenshot_2](https://user-images.githubusercontent.com/36493429/180831844-9704f3da-9f00-4a72-bba3-41f3a184b82c.png)

**Reply by Snailedlt** [reply]
@IReena8 I'm 90% sure this is a newly introduced bug after a GitHub update a few hours ago. Best you can do is wait for them to fix it.

In the meantime, link them to the folder that the file is in instead, then tell them to navigate to the file.

**Reply by MrSarno** [reply]
I'm terribly sorry, @IReena8, I'd misread the URL earlier. :(

I realise now that it was a branch. Are you able to access this file without issue now?

https://github.com/IReena8/GitBash/blob/Postman/Group_30.postman_environment.json

**Comment by ghost** [comment]
this article helpful
https://kinsta.com/blog/406-error/#:~:text=The%20primary%20way%20to%20address%20and%20fix%20a,Headers%20to%20reveal%20all%20requests%20from%20this%20webpage.

**Reply by MrSarno** [reply]
That article has advice for people operating their own site.

The discussion is about GitHub throwing the OP that error message. She is not in a position to make changes to GitHub itself.

**Reply by ghost** [reply]
@MrSarno oh sorry, I have think this will help her too

**Reply by ghost** [reply]
@MrSarno 
ok look here 
https://stackoverflow.com/questions/11577147/how-to-fix-http-404-on-github-pages

**Reply by ghost** [reply]
@MrSarno and here 
https://stackoverflow.com/questions/67135270/getting-406-not-acceptable-when-trying-to-download-a-git-repo-using-powershe#:~:text=GitHub%20is%20responding%20406%20Unacceptable%20because%20it%20don%27t,modify%20your%20header%20like%20this%3A%20Accept%20%3D%20%22application%2Fvnd.github.v3.raw%2C%2A%22

**Reply by Snailedlt** [reply]
@GAbdulWahab Sadly none of those are relevant either. 

It's a bug inteoduced by a recent GitHub update. So not much we can do other than wait for GitHub to fix it (or manually navigate to the file via the GitHub UI).

**Comment by ghost** [comment]


**Reply by MrSarno** [reply]
As I said in response to your other comment, OP does not work for GitHub and cannot make changes to the site.

Her issue is caused by files seemingly not having been pushed to GitHub. Please don’t spam unhelpful links.

**Reply by ghost** [reply]
ok

**Reply by IReena8** [reply]
Thank you so much !!!! for your help!!!

**Reply by IReena8** [reply]
Thank you so much !!!! for your help!!!

**Comment by Snailedlt** [comment]
I get the same issue when trying to access a javascript file. I think the error has to do with the file extension, and a recent update either to chrome or GitHub

If I open the console it says something about not accepting javascript. I will copy both the link, and the error when I get back to my PC

**Comment by IReena8** [comment]
@MrSarno  would you  send me link with protection of branch that you sent me in first answer? Because I cant find out this page with Protection((   I seem I pressed extra checkboxes  and I'm afraid . I need to check!

**Reply by Snailedlt** [reply]
I think they fixed the issue now. I'm not getting the 406 error anymore
json -> https://github.com/IReena8/GitBash/blob/main/HW_Postman_1.postman_collection.json
js -> https://github.com/svg/svgo/blob/main/plugins/inlineStyles.js

**Comment by IReena8** [comment]
Yeeees! Thank you my friends for your support!!!!!

**Comment by Snailedlt** [comment, marked as answer]
The issue has been fixed now! 🎉 
json -> ~~https://github.com/IReena8/GitBash/blob/main/HW_Postman_1.postman_collection.json~~ https://github.com/IReena8/GitBash/blob/Postman/Group_30.postman_environment.json
js -> https://github.com/svg/svgo/blob/main/plugins/inlineStyles.js

@IReena8 please mark this as the answer so others can see as well :)

**EDIT:**
Like @MrSarno kindly noted before, the file I linked to was the wrong one. I can't find out now if it worked all along or not, but it at least works now. I still think this is a relevant answer though, since the update which was recently pushed fixed the 406 error bug when trying to access .js files, which I assume is the same bug.

**Reply by MrSarno** [reply]
Respectfully, I'm having difficulty understanding why you asked for this to be marked as the answer - that JSON URL was accessible when I first linked to it 3.5 hours prior to you, and it's not the URL  OP had been attempting to access (more coffee needed on my part!).

**Reply by Snailedlt** [reply]
@MrSarno Whoopsie! You're absolutely right!
I just assumed this was the right file since I was browsing through the repo and that was the only .json file I found on the main branch. No wonder I couldn't find it though, since it was on the `Postman` branch, and not the `main` branch.

I guess it's me who needs some coffee ehh?
Anyways... My answer has been updated to address this.

**Reply by MrSarno** [reply]
@Snailedlt I won't tell if you won't. :P
